.github/workflows/node.yml

```diff
@@ -177,3 +177,39 @@ jobs:
             - name: Test compatibility
               run: npm test -- -t "set and get flow works"
               working-directory: ./node
+
+
+    build-linux-musl-latest:
+        runs-on: ubuntu-latest
+        container:
+            image: node:18.19.0-alpine
+        timeout-minutes: 15
+        steps:
+            - name: Install git
+              run: apk add git
+
+            - uses: actions/checkout@v4
+
+            - name: Checkout submodules
+              run: |
+                  git config --global --add safe.directory "$GITHUB_WORKSPACE"
+                  git submodule update --init --recursive
+
+            - name: Build Node wrapper
+              uses: ./.github/workflows/build-node-wrapper
+              with:
+                  os: "ubuntu-latest-musl"
+                  target: "aarch64-unknown-linux-musl"
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+                  arch: "arm64"
+                  named_os: "linux-musl"
```

Comment:
We should keep node_os in node-create-package file as 'linux', so you might want to create a new optional 'libc' argument that will hold the 'musl' suffix

Reply:
That'll probably work.  
As I see it,   it the only current solution,  if we're getting rid of the `existsSync` (and we should).
There is another direction we can take,   I'll explain further below.